ChangeLog.md

```diff
@@ -8,14 +7,22 @@
 * Implemented hardware cursor support
 * Heroes III language can now be detected automatically
 * Increased targeted framerate from 48 to 60
+* Increased performance of UI updates
 * Fixed bonus values of heroes who specialize in secondary skills
 * Fixed bonus values of heroes who specialize in creatures
 * Fixed damage increase from Adela's Bless specialty
 * Fixed missing obstacles in battles on subterranean terrain 
 * Added vcmifaerie, vcmiazure, vcmiarmy, vcmiexp cheats
 * Video files now play at correct speed
 * Fixed crash on switching to second mission in campaigns
+* New cheat code: vcmiarmy or vcminissi - gives specified creatures in every empty slot
+* New cheat code: vcmiexp or vcmiolorin - gives specified amount of experience to current hero

```

Comment:
* Video files now play at correct speed
* Fixed crash on switching to second mission in campaigns
* New cheat code: vcmiazure - give 5000 azure dragons in every empty slot
* New cheat code: vcmifaerie - give 5000 faerie dragons in every empty slot
* New cheat code: vcmiarmy or vcminissi - give specified creatures in every empty slot. EG: vcmiarmy imp
* New cheat code: vcmiexp or vcmiolorin - give specified amount of experience to current hero. EG: vcmiexp 10000